### PR TITLE
edit strings for consistency, branding (AWS)

### PR DESCRIPTION
**Pull request by jamescott**
Consistent converge_by statements
Consistent capitalization
Applied AWS branding patterns to all terms

### CODE REVIEW

File: lib/chef/provider/aws_auto_scaling_group.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Chef::Provider::AwsAutoScalingGroup < Chef::Provisioning::AWSDriver::AWSPr
   protected
 
   def create_aws_object
-    converge_by "create new Auto Scaling Group #{new_resource.name} in #{region}" do
+    converge_by "create Auto Scaling group #{new_resource.name} in #{region}" do
```
Comment (dblessing) [inline lib/chef/provider/aws_auto_scaling_group.rb]: I love the consistency of changing all the strings to match. I think we should use capitalization whenever the message directly references something in the AWS console. For example, here it should be 'Auto Scaling Group' as capitalized.

Comment (tpetchel) [inline lib/chef/provider/aws_auto_scaling_group.rb]: I think we should call it whatever Amazon calls it. For example, "Auto Scaling group".
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html

**Comment by randomcamel** [comment]
Looks great! Let's chat briefly with folks tomorrow to identify a standard for us to use going forward, then merge this.

File: lib/chef/provider/aws_auto_scaling_group.rb (hunk 1 above)
Comment (tyler-ball) [inline lib/chef/provider/aws_auto_scaling_group.rb]: Even the console is divided on this - sometimes I see `Group` and sometimes I see `group`.  But it looks like the `G` is only in titles and navigation bars, which typically use title casing anyways.  I'm declare it shall be `g`roup!

**Comment by tyler-ball** [comment]
Thanks @jamescott !

**Pull request merged**